tests/unit/server/api/dataloaders/conftest.py

```diff
@@ -14,80 +14,98 @@ async def data_for_testing_dataloaders(
 ) -> None:
     seed(42)
     orig_time = datetime.fromisoformat("2021-01-01T00:00:00.000+00:00")
-    I, J, K = 10, 10, 10  # noqa: E741
+    num_projects = 10
+    num_sessions_per_project = 2
+    num_traces_per_session = 5
+    num_spans_per_trace = 10
     async with db() as session:
-        for i in range(I):
+        for i in range(num_projects):
             project_row_id = await session.scalar(
                 insert(models.Project).values(name=f"{i}").returning(models.Project.id)
             )
-            for j in range(J):
+            for l in range(num_sessions_per_project):  # noqa: E741
                 seconds = randint(1, 1000)
                 start_time = orig_time + timedelta(seconds=seconds)
-                end_time = orig_time + timedelta(seconds=seconds * K * 2)
-                trace_row_id = await session.scalar(
-                    insert(models.Trace)
+                end_time = orig_time + timedelta(seconds=seconds * l * 2)
+                session_row_id = await session.scalar(
+                    insert(models.ProjectSession)
                     .values(
-                        trace_id=f"{i}_{j}",
-                        project_rowid=project_row_id,
+                        session_id=f"{i}_{l}",
```

Comment:
nit. I was lazy, but you don't have to be. lol
```suggestion
                        session_id=f"proj{i}_sess{l}",
```